### PR TITLE
fix: refactor home tab to use native-base

### LINKED ISSUE #24: FRONTEND: Refactor Home tab to use Native Base

### PR DESCRIPTION
**Pull request by dffattal**
Closes #24

### CODE REVIEW

File: app/views/homeTab/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    const formState = this.state.showForm
+    return (
+      <Drawer
+        ref={ref => {this.sidebar = ref}}
+        content={<Sidebar navigator={this.navigator} />}
+        styles={{drawer: {backgroundColor: "white"}}}
+        onClose={this.closeSidebar} >
+        <Container>
+          <Content>
+            <Text style={getTheme()['NativeBase.Title']}>My Groups</Text>
```
Comment (JeffKandel) [inline app/views/homeTab/components/Home.js]: I doesn't make sense to have to getTheme every single time we want to style a component.  Could we ask Chaim about this?

File: app/views/homeTab/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  openSidebar() {
+    this.sidebar._root.open()
+  }
+  closeSidebar() {
+    this.sidebar._root.close()
+  }
+  render() {
+    const formState = this.state.showForm
+    return (
+      <Drawer
```
Comment (JeffKandel) [inline app/views/homeTab/components/Home.js]: I don't love the drawer.  I think User Profile info could be it's own screen. Let's check what AirBnB does

**Review by JeffKandel — COMMENTED** [review]
Hey danny, I think we can do this a little cleaner.

File: app/views/homeTab/components/Home.js (hunk 1 above)
Comment (dffattal) [reply app/views/homeTab/components/Home.js]: The Title component doesn't seem to render unless we wrap it inside a Header and Body.

**Review by JeffKandel — APPROVED** [review]
Please merge

**Pull request merged**